.ahoy.yml

```diff
@@ -36,6 +36,10 @@ commands:
     usage: Build and start Docker containers.
     cmd: |
       echo "Preparing containers for $VARS_TYPE"
+      CKAN_VERSION=$(sh retrieve-ckan-version.sh)
```

Comment:
I guess next step after this is python version 🙃

Reply:
Once CKAN 2.9 is all sorted, I hope so :)